package.json

```diff
@@ -43,6 +43,7 @@
     "rimraf": "^5.0.1",
     "rxjs": "^7.8.1",
     "semver": "^7.5.4",
+    "viem": "^1.5.3",
```

Comment:
nitpick: If you want to upgrade `viem` here to `1.7.0` since this is the first PR to introduce it. Else we can do it in a separate PR.